helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/FieldBuilder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Field.Order;
+
+
+/**
+ * an builder interface to build a Schema.Field with given parameters.
+ * The defaultValue needed to be defined in implemented classes since default
+ * value type is changed to Object since AVRO 1.9
```

Comment:
we could support getting/setting the default value as either Object (primitive-wrapper collection or GenericData.Record), and/or as a json string (cat least as a todo)